src/create_character.cpp

```diff
@@ -147,7 +181,7 @@ void PickBgState::update()
 
     case MenuAction::esc:
     {
-      pop_states(4);
+      pop_until(states::StateId::MENU_STATE);
```

Comment:
I forgot about the namespace - but it didn't complain when compiling and it ran fine. If I remove the namespace I'll keep this as it is.